op-challenger/fault/cannon/provider.go

```diff
@@ -82,8 +84,19 @@ func (p *CannonTraceProvider) GetPreimage(ctx context.Context, i uint64) ([]byte
 	return value, data, nil
 }
 
-func (p *CannonTraceProvider) AbsolutePreState(ctx context.Context) []byte {
-	panic("absolute prestate not yet supported")
+func (p *CannonTraceProvider) AbsolutePreState(ctx context.Context) ([]byte, error) {
+	path := filepath.Join(p.dir, elfState)
```

Comment:
This should be coming from the config options - `config.CannonAbsolutePreState`. We have the config in `NewCannonTraceProvider` so just need to grab the value on the way through (currently it just passes everything down to the executor).